pages/form-builder/responses/[[...params]].tsx

```diff
@@ -47,12 +72,87 @@ const Responses: NextPageWithLayout<ResponsesProps> = ({ vaultSubmissions }: Res
     <Button onClick={dialogReportProblemsHandleClose}>{t("responses.reportProblems")}</Button>
   );
 
+  // Note: A hack to enable an ajax file download through axios. Oddly setting the responseType
+  // is not enough. Come back to this, suspect there is a better way..
+  const forceFileDownload = (response: AxiosResponse, fileName: string) => {
+    // NOTE: intentionally allowing Errors to be thrown and caught in container Promise
```

Comment:
We have more examples of his way of downloading in ShareModelUnauthenticated, DownloadFileButton and DowloadCSV 

- Let's remove the "hack" comment above here and maybe add add a file type param like the others.